CA_DataUploaderLib/ServerUploader.cs

```diff
@@ -258,7 +255,7 @@ private void LoopForever()
                     }
 
                     PrintBadPackagesMessage(false);
-                    Thread.Sleep(MillisecondsBetweenUpload);  // only send approx. one time per second. 
+                    Thread.Sleep(IOconfFile.GetVectorUploadDelay());  

```

Comment:
probably something for a separate PR, but this sleep should consider how time it already passed since the last upload i.e. to keep the desired frequency.